packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2063,6 +2063,10 @@ export default mixins(
 				const resData = await this.importWorkflowData(data.data as IWorkflowDataUpdate);
 			});
 
+			this.$root.$on('newWorkflow', () => {
+				this.newWorkflow();
+			});
```

Comment:
This should work as well

```suggestion
			this.$root.$on('newWorkflow', this.newWorkflow);
```

Reply:
Nice suggestion, applied!